src/components/VolumeIndicator/VolumeIndicator.vue

```diff
@@ -129,17 +133,15 @@ export default {
 				return 0
 			}
 
-			return (this.size - this.iconOffsetBottom) * this.computeVolumeLevel()
+			return (this.iconSizeWithoutOffset) * this.computeVolumeLevel()
 		},
 	},
 
 	methods: {
-		computeVolumeLevel() {
+		computeVolumeLevel(): number {
```

Comment:
Note, TS can determine types in many cases. Explicit type declarations is needed when it cannot be computed and on public interface to specify a contract.